pkg/userdata/helper/helper.go

```diff
@@ -102,17 +102,22 @@ SystemMaxUse=5G
 }
 
 type dockerConfig struct {
-	ExecOpts           []string `json:"exec-opts"`
-	StorageDriver      string   `json:"storage-driver"`
-	InsecureRegistries []string `json:"insecure-registries"`
-	RegistryMirrors    []string `json:"registry-mirrors"`
+	ExecOpts           []string          `json:"exec-opts,omitempty"`
+	StorageDriver      string            `json:"storage-driver,omitempty"`
+	StorageOpts        []string          `json:"storage-opts,omitempty"`
```

Comment:
Where this field is used?

Reply:
it was intended for RHEL, but since it works without it it's not used currently